webview-ui/src/i18n/TranslationContext.tsx

```diff
@@ -46,7 +46,10 @@ export const TranslationProvider: React.FC<{ children: ReactNode }> = ({ childre
 				t: translate,
 				i18n,
 			}}>
-			{children}
+			{/* kilocode_change: add lang, dir */}
+			<div lang={i18n.language} dir={i18n.dir()}>
```

Comment:
This is a potentially impactful change, because it wraps everything in an extra div, but I haven't noticed any problems.